### ISSUE TITLE
UnicodeEncodeError when sys.stdout.encoding is ISO-8859-1

### ISSUE THREAD

**Issue by LinuxMercedes** [Rockhopper-Technologies/enlighten#13]
I ran into an issue where on a Linux system with the terminal encoding set to ISO-8859-1, I get the following `UnicodeEncodeError`:

```
  File "/<snip>/.local/lib/python3.6/site-packages/enlighten/_counter.py",
line 736, in update
    self.refresh(elapsed=currentTime - self.start)
  File "/<snip>/.local/lib/python3.6/site-packages/enlighten/_counter.py",
line 716, in refresh
    flush=flush, position=self.position)
  File "/<snip>/.local/lib/python3.6/site-packages/enlighten/_manager.py",
line 372, in write
    stream.write(u'\r' + term.clear_eol + output)
UnicodeEncodeError: 'latin-1' codec can't encode characters in
position 9-10: ordinal not in range(256)
```

I'm not entirely sure how to handle this; I believe ISO-8859-1 only has ASCII characters so `|` is the only thing that could really be used for a progress bar.

EDIT: remove misdiagnosis

**Comment by avylove** [comment]
Thanks for reporting.
What version of Python are you using?
Do you get the error if you pass `series=' |'` as argument when creating the progress bar?

**Comment by avylove** [comment]
Also what is the output of this?
```
$ python3 -c 'import sys; print(sys.__stdout__.encoding)'
```

**Comment by LinuxMercedes** [comment]
Yep, that's the workaround I'm using for now and it works just fine.

The python snippet returns `ISO-8859-1`.

**Comment by avylove** [comment]
I'll see if I can come up with a good way to detect and use an alternative default bar. For your workaround you can also try `series=' ¦|'` or `series=' ·oO')` which may look a little more visually appealing.

**Comment by avylove** [comment]
I put a fix in 9329164612238fac36fd331e38754a70c4b499ec. Please test it out and let me know if it works for you.

**Comment by LinuxMercedes** [comment]
This seems to work for me! Thanks for putting in a fix for a pretty niche corner case.

**Comment by avylove** [comment]
No problem. I still have a couple things I need to do before the next release. I'll try not to take too long.

**Comment by avylove** [comment]
Released with [1.5.0](https://github.com/Rockhopper-Technologies/enlighten/releases/tag/1.5.0)
